client/packages/common/src/ui/forms/JsonForms/components/Group.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { rankWith, uiTypeIs, LayoutProps, GroupLayout } from '@jsonforms/core';
+import { withJsonFormsLayoutProps } from '@jsonforms/react';
+import { MaterialLayoutRenderer } from '@jsonforms/material-renderers';
+import { Box, Typography } from '@mui/material';
+
+export const groupTester = rankWith(4, uiTypeIs('Group'));
+
+const UIComponent = (props: LayoutProps) => {
+  const { uischema, schema, visible, renderers, path } = props;
+
+  const layoutProps = {
+    elements: (uischema as GroupLayout).elements,
+    schema: schema,
+    path: path,
+    direction: 'column' as 'column' | 'row',
+    visible: visible,
+    uischema: uischema,
+    renderers: renderers,
+  };
+  return (
+    <Box
+      sx={{
+        maxWidth: 500,
```

Comment:
the maxWidth is fairly specific, just wondering if that will cause problems later.

Reply:
I've moved these values into a "styleConstants" file so we can update them easily if we need to.

Also -- if we need a wider "Group" in future, we can always make "width" a new property in uiSchema which the component can use, or have an alternative component "WideGroup". (for one that spans 2 columns, perhaps?)